source-browser.html

```diff
@@ -181,7 +209,46 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
           .focus();
       }
 
+      function createNavbar(topic) {
+        $('.li').append(
+          $('<a>', {
+            id: `li${topic}`,
+            href: `#${topic}`
+          }).text(topic)
+        );
+      }
+
+      let list = [];
+      $(document).ready(() => {
+	    TOPICS.forEach(item => {
+	      let heading = item.name;
+	      list.push(heading);
+	    });
+      });
+
+      $(window).scroll(() => {
+        let top = $(window).scrollTop();
+        let bottom = $(window).scrollTop() + $(window).height();
+        list.forEach(heading => {
+          let vartop = $(`#${heading}`).position();
```

Comment:
What does the `var` in `vartop` refer to?

Reply:
It refers to variable, for each of the heading variables.  Would to suggest I rename it?

Reply:
Does it make sense to name a variable `var` to indicate that it's a variable? Something like `heading_position` maybe.

Reply:
Okay, I'll rename that.